hyperstart.go

```diff
@@ -296,6 +296,24 @@ func (h *hyper) bindMountContainerMounts(podID string, c Container, mounts []Mou
 	return fsmap, nil
 }
 
+func (h *hyper) bindUnmountContainerMounts(podID string, c *Container) error {
```

Comment:
This is a container generic routine and should be implemented as a private `Container` method instead. Again, we want that one to be usable by all potential hypervisor implementations.